wdfn-server/waterdata/templates/partials/header.html

```diff
@@ -88,35 +117,20 @@
                             </a>
                         </li>
                         <li class="usa-nav__submenu-item">
-                            <a href="https://www.usgs.gov/special-topic/water-science-school"
-                               rel="noopener" target="_blank"
-                               ga-on="click" ga-event-category="navigation" ga-event-action="WaterScienceSchoolNav">
-                                How Data are Collected
-                            </a>
+                            <a href="https://waterservices.usgs.gov/">USGS Water Data APIs</a>
```

Comment:
Does this one need the target and other attributes?

Reply:
It is our product so I am assuming not and we didn't before. This one was just picked up and moved.